modules/ROOT/pages/techinfo/buildsystem.adoc

```diff
@@ -10,13 +10,50 @@ This page covers the CentOS Stream Build System.
 
 == Module Build Service 
 
+=== Process for module building in Stream 9
+
+* Make sure you have the latest version of `centpkg` installed--0.6.6 or later is required.
+
+* If your module does not yet exist in the
+https://gitlab.com/redhat/centos-stream/modules[CentOS Stream modules namespace in GitLab],
+open a https://issues.redhat.com/secure/CreateIssue!default.jspa?issuetype=3&pid=12326233[CS Jira]
```

Comment:
I thought that the default place for the issues should be bugzilla.

Reply:
> I thought that the default place for the issues should be bugzilla.

Sure. That makes sense.

@bstinsonmhk said he had some thoughts about about filters and/or keywords to make sure such a BZ is directed to the proper queue. I imagine product="Red Hat Enterprise Linux 9" and version="CentOS Stream" is the starting point. What else?